streams/shared/src/main/scala/zio/stream/ZStream.scala

```diff
@@ -1641,6 +1641,18 @@ class ZStream[-R, +E, +A] private[stream] (private[stream] val structure: ZStrea
   final def mapMParUnordered[R1 <: R, E1 >: E, B](n: Int)(f: A => ZIO[R1, E1, B]): ZStream[R1, E1, B] =
     self.flatMapPar[R1, E1, B](n)(a => ZStream.fromEffect(f(a)))
 
+  /**
+   * Maps over elements of the stream with the specified effectful function,
+   * partitioned by `p` executing invocations of `f` concurrently on up
+   * to `buffer` partitions. Transformed elements may be reordered but the order within
```

Comment:
One last comment: `buffer` does not prescribe how many partitions; that is prescribes by however many different values of `K` are returned from `keyBy`.

The buffer describes the amount of elements that can be buffered in each partition.

Reply:
Thanks, good catch. I have updated this to be more correct about the buffer and tried to be more explicit about how the partition function affects how many concurrent invocations of `f` there are.